test/NinjaTools.FluentMockServer.API.Tests/Models/RequestMatcherTests.cs

```diff
@@ -1,249 +1,251 @@
-            
-            // Act & Assert
-            subject.IsMatch(context).Should().BeFalse();
-        }
-
-        [Fact]
-        public void IsMatch_Should_Be_False_When_RequestBodyMatcher_IsNotMatch()
-        {
-            // Arrange
-            var requestBodyContent = "some content";
-            var requestBodyStream = new MemoryStream();
-            var bytes = Encoding.UTF8.GetBytes(requestBodyContent);
-            requestBodyStream.Write(bytes);
-            var bodyMatcher = new RequestBodyMatcher
-            {
-                Content = "hello world",
-                Type = RequestBodyType.Text,
-                MatchExact = true
-            };
-            
-            var subject = CreateSubject(bodyMatcher: bodyMatcher );
-            var context = CreateContext(requestBodyStream: requestBodyStream);
-            
-            // Act & Assert
-            subject.IsMatch(context).Should().BeFalse();
-        }
-
-
-        [Fact]
-        public void IsMatch_Should_Be_True_When_No_RequestBodyMatcher()
-        {
-            // Arrange
-            var subject = CreateSubject();
-            var context = CreateContext();
-            
-            // Act & Assert
-            subject.IsMatch(context).Should().BeTrue();
-        }
-
-        [Fact]
-        public void IsMatch_Should_Be_True_When_QueryString_IsMatch()
-        {
-            // Arrange
-            var query = new QueryString("?id=100");
-            var subject = CreateSubject(queryString: query);
-            var context = CreateContext(queryString: query);
-            
-            // Act & Assert
-            subject.IsMatch(context).Should().BeTrue();
-        }
-
-        [Fact]
-        public void IsMatch_Should_Be_True_When_QueryString_Not_Set()
-        {
-            // Arrange
-            var query = new QueryString("?id=100");
-            var subject = CreateSubject();
-            var context = CreateContext(queryString: query);
-            
-            // Act & Assert
-            subject.IsMatch(context).Should().BeTrue();
-        }
-
-
-        [Fact]
-        public void IsMatch_Should_Be_True_When_RequestBodyMatcher_IsMatch()
-        {
-            // Arrange
-            var requestBodyContent = "some content";
-            var requestBodyStream = new MemoryStream();
-            var bytes = Encoding.UTF8.GetBytes(requestBodyContent);
-            requestBodyStream.Write(bytes);
-            var bodyMatcher = new RequestBodyMatcher
-            {
-              Content = requestBodyContent,
-              Type = RequestBodyType.Text,
-              MatchExact = true
-            };
-            
-            var subject = CreateSubject(bodyMatcher: bodyMatcher );
-            var context = CreateContext(requestBodyStream: requestBodyStream);
-            
-            // Act & Assert
-            subject.IsMatch(context).Should().BeTrue();
-        }
-    }
-}
+// using System.Collections.Generic;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)